### PR TITLE
GEODE-4795: User Guide: Inequality queries return UNDEFINED entries

### PR DESCRIPTION
**Pull request by davebarnes97**
Added statement that inequality queries return UNDEFINED entries.
Did some reformatting while I was editing the page.

### CODE REVIEW

File: geode-docs/developing/query_additional/operators.html.md.erb [hunk 1]
```diff
@@ -27,25 +27,31 @@ Comparison operators compare two values and return the results, either TRUE or F
...
+|          |                          |
+| =        | equal to                 |
+| !=       | not equal to             |
+| &lt;&gt; | not equal to             |
 
-The equal and not equal operators have lower precedence than the other comparison operators. They can be used with null. To perform equality or inequality comparisons with UNDEFINED, use the IS\_DEFINED and IS\_UNDEFINED preset query functions instead of these comparison operators.
+Regarding equality and inequality operators:
+
+- The equal and not equal operators have lower precedence than the other comparison operators.
+- The equal and not inequal operators can be used with null.
```
Comment (joeymcallister) [inline geode-docs/developing/query_additional/operators.html.md.erb]: Should this be "The equal and not-equal operators"?

File: geode-docs/developing/query_additional/operators.html.md.erb [hunk 2]
```diff
@@ -27,25 +27,31 @@ Comparison operators compare two values and return the results, either TRUE or F
...
 
 ## <a id="operators__section_A970AE75B0D24E0B9E1B61BE2D9842D8" class="no-quick-link"></a>Unary Operators
 
-Unary operators operate on a single value or expression, and have lower precedence than comparison operators in expressions. <%=vars.product_name%> supports the unary operator NOT. NOT is the negation operator, which changes the value of the operand to its opposite. So if an expression evaluates to TRUE, NOT changes it to FALSE. The operand must be a boolean.
+Unary operators operate on a single value or expression, and have lower precedence than comparison operators in expressions. <%=vars.product_name%> supports the unary operator NOT. NOT is the negation operator, which changes the value of the operand to its opposite, so if an expression evaluates to TRUE, NOT changes it to FALSE. The operand must be a boolean.
```
Comment (joeymcallister) [inline geode-docs/developing/query_additional/operators.html.md.erb]: I agree with not starting a new sentence with "so," but I also see why the original writer may have  wanted to break up the sentence. What do you think about "... to its opposite. For example, if an expression ..."?

File: geode-docs/developing/query_additional/operators.html.md.erb [hunk 3]
```diff
@@ -57,20 +63,20 @@ and `getCause()` will state `ArithmeticException`.
...
 
 Map and index operators access elements in key/value collections (such as maps and regions) and ordered collections (such as arrays, lists, and `String`s). The operator is represented by a set of square brackets (`[ ]`) immediately following the name of the collection. The mapping or indexing specification is provided inside these brackets.
 
-Array, list, and `String` elements are accessed using an index value. Indexing starts from zero for the first element, 1 for the second element and so on. If `myList` is an array, list, or String and `index` is an expression that evaluates to a non-negative integer, then `myList[index]` represents the (`index + 1`)th element of `myList`. The elements of a `String` are the list of characters that make up the string.
+Array, list, and `String` elements are accessed using an index value. Indexing starts from zero for the first element, 1 for the second element and so on. If `myList` is an array, list, or `String` and `index` is an expression that evaluates to a non-negative integer, then `myList[index]` represents the (`index + 1`)th element of `myList`. The elements of a `String` are the list of characters that make up the string.
```
Comment (joeymcallister) [inline geode-docs/developing/query_additional/operators.html.md.erb]: Add a serial comma after "second element."

File: geode-docs/developing/query_additional/operators.html.md.erb (hunk 1 above)
Comment (echobravopapa) [reply geode-docs/developing/query_additional/operators.html.md.erb]: equality and inequality operators I think is the term we are looking for here..

**Comment by davebarnes97** [comment]
I've received some constructive comments outside this PR thread, and have also found at least one other location where this material needs to be repeated (the "Supported Keywords" table).
There's enough additional work to be done that I'm closing this PR.  I will resubmit later.

**Pull request closed without merging**